src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -718,9 +748,9 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
     private static List<ExceptionInfo> combineExceptionInfo(List<ExceptionInfo> list1,
                                                      List<ExceptionInfo> list2) {
         final List<ExceptionInfo> result = new ArrayList<>(list1);
-        for (ExceptionInfo expectionInfo : list2) {
-            if (result.stream().noneMatch(item -> isExceptionInfoSame(item, expectionInfo))) {
-                result.add(expectionInfo);
+        for (ExceptionInfo exceptionInfo : list2) {
+            if (result.stream().noneMatch(item -> isExceptionInfoSame(item, exceptionInfo))) {
+                result.add(exceptionInfo);
```

Comment:
This PR will not be simple and quick.
Please move  this already separated commit to separate PR, we will merge it quicker. So it will help us to keep this PR more focused on functional changes.

Reply:
Done #8114